django-stubs/utils/deprecation.pyi

```diff
@@ -29,10 +30,19 @@ class DeprecationInstanceCheck(type):
     deprecation_warning: type[Warning]
     def __instancecheck__(self, instance: Any) -> bool: ...
 
-class GetResponseCallable(Protocol):
+@type_check_only
```

Comment:
We generally don't use `@type_check_only` for protected protocols.

```suggestion
```

Let's do it consistently.